shibboleth-cluster-windows/azuredeploy.json

```diff
@@ -0,0 +1,648 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "User name for the Virtual Machine administrator. Do not use simple names such as 'admin'."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine administrator."
+      }
+    },
+    "namePrefix": {
```

Comment:
Prefix this with unique to set the right expectations otherwise users will run into non-unique DNS name failures with common inputs like "test", "my" etc.